templates/fragments/meta.html

```diff
@@ -1,9 +1,25 @@
+<meta property="og:title" content="{{ meta.title.content }}">
+<meta name="twitter:title" content="{{ meta.title.content }}">
 {% if meta.description.content %}
 <meta name="description" content="{{ meta.description.content }}">
+<meta property="og:description" content="{{ meta.description.content }}">
+<meta name="twitter:description" content="{{ meta.description.content }}">
+{% endif %}
+{% if meta.canonical_url %}
+<meta property="og:url" content="{{ meta.canonical_url }}">
+<meta name="twitter:url" content="{{ meta.canonical_url }}">
 {% endif %}
 {% if meta.keywords.content %}
 <meta name="keywords" content="{{ meta.keywords.content }}">
 {% endif %}
+{% if meta.image %}
+<meta property="og:image" content="{{ meta.image }}">
+<meta name="twitter:image" content="{{ meta.image }}">
+<meta name="twitter:card" content="summary_large_image">
+{% else %}
+<meta name="twitter:card" content="summary">
+{% endif %}
```

Comment:
```suggestion
<meta name="twitter:card" content="summary{% if meta.image %}_large_image{% endif %}">
```

- Combine the two cases. One line per tag is more readable IMO
- Note the no extra spaces